src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -84,19 +79,17 @@ public void ejectGamePiece() {
   }
 
   public void clampIntake() {
-    clampLeft.set(DoubleSolenoid.Value.kForward);
-    clampRight.set(DoubleSolenoid.Value.kForward);
+    clamp.set(false);
   }
 
   public void unclampIntake() {
-    clampLeft.set(DoubleSolenoid.Value.kReverse);
-    clampRight.set(DoubleSolenoid.Value.kReverse);
+    clamp.set(true);
   }
 
   @Override
   public void periodic() {
     if (clampTimer.hasElapsed(Calibrations.AUTO_CLAMP_WAIT_TIME_SECONDS)) {
-      clampLeft.set(DoubleSolenoid.Value.kForward);
+      clamp.set(false);
```

Comment:
Can you call `clampIntake()` instead of `clamp.set(false)`. The latter it's a little less clear what it's doing.